src/sqlfluff/core/linter/linted_file.py

```diff
@@ -205,15 +205,16 @@ def _log_hints(
     ):
         """Log hints for debugging during patch generation."""
         # This next bit is ALL FOR LOGGING AND DEBUGGING
-        if patch.templated_slice.start >= 10:
+        max_log_length = 10
```

Comment:
Unrelated to the fix... this change makes it easier to tweak the max size of log hints for patches. Larger size can be helpful for debugging some issues.